frostsnapp/lib/main.dart

```diff
@@ -196,7 +197,23 @@ class _MyHomePageState extends State<MyHomePage> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-        appBar: AppBar(title: Text(widget.title)),
+        appBar: AppBar(title: Text(widget.title), actions: [
```

Comment:
This should not just be on the home page. It should be on every page. Make a custom appbar widget which always has the action set and passes the widget title to the feedback form submission as well so we know where they were when they clicked it.